VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control.service.spec.ts

```diff
@@ -0,0 +1,361 @@
+import { HearingRole } from 'src/app/waiting-space/models/hearing-role-model';
+import { ParticipantUpdated } from 'src/app/waiting-space/models/video-call-models';
+import { VideoCallService } from 'src/app/waiting-space/services/video-call.service';
+import { ParticipantForUserResponse, ParticipantStatus, Role } from '../clients/api-client';
+import { LoggerService } from '../logging/logger.service';
+import { ConferenceService } from './conference.service';
+import { PexipDisplayNameModel } from './models/pexip-display-name.model';
+import { VirtualMeetingRoomModel } from './models/virtual-meeting-room.model';
+import { IHearingControlsState, VideoControlCacheService } from './video-control-cache.service';
+import { VideoControlService } from './video-control.service';
+
+describe('VideoControlService', () => {
+    let conferenceServiceSpy: jasmine.SpyObj<ConferenceService>;
+    let videoCallServiceSpy: jasmine.SpyObj<VideoCallService>;
+
+    let videoControlCacheServiceSpy: jasmine.SpyObj<VideoControlCacheService>;
+    let loggerSpy: jasmine.SpyObj<LoggerService>;
+
+    let sut: VideoControlService;
+
+    const conferenceId = 'conference-id-1';
+    const participantOneId = Guid.create().toString();
+    const participantOnePeixpId = Guid.create().toString();
+    const participantOne = ParticipantModel.fromParticipantForUserResponse(
+        new ParticipantForUserResponse({
+            id: participantOneId,
+            status: ParticipantStatus.NotSignedIn,
+            display_name: 'Interpreter',
+            role: Role.Individual,
+            representee: null,
+            case_type_group: 'applicant',
+            tiled_display_name: `CIVILIAN;Interpreter;${participantOneId}`,
+            hearing_role: HearingRole.INTERPRETER,
+            first_name: 'Interpreter',
+            last_name: 'Doe',
+            interpreter_room: null,
+            linked_participants: []
+        })
+    );
+
+    beforeEach(() => {
+        participantOne.pexipId = participantOnePeixpId;
+
+        conferenceServiceSpy = jasmine.createSpyObj<ConferenceService>('ConferenceService', ['getConferenceById'], ['currentConferenceId']);
+        getSpiedPropertyGetter(conferenceServiceSpy, 'currentConferenceId').and.returnValue(conferenceId);
+
+        videoCallServiceSpy = jasmine.createSpyObj<VideoCallService>('VideoCallService', ['spotlightParticipant', 'onParticipantUpdated']);
+
+        videoControlCacheServiceSpy = jasmine.createSpyObj<VideoControlCacheService>('VideoControlCacheService', [
+            'setSpotlightStatus',
+            'getSpotlightStatus',
+            'getStateForConference'
+        ]);
+
+        loggerSpy = jasmine.createSpyObj<LoggerService>('Logger', ['error', 'warn', 'info']);
+
+        sut = new VideoControlService(conferenceServiceSpy, videoCallServiceSpy, videoControlCacheServiceSpy, loggerSpy);
+    });
+
+    it('should be created', () => {
+        expect(sut).toBeTruthy();
```

Comment:
same as the other spec files, I would recommend removing this test as it would get covered by other unit tests in this file